contrib/localnet/deposit-test.sh

```diff
@@ -50,7 +50,7 @@ done
 
 #
 # wait for proposal to pass
-for i in {0..6}; do
+for i in {0..7}; do
```

Comment:
More magic numbers :P Source from ENV variables, default to desired "sensible default"